src/Dropdown/test/DropdownMenuSpec.js

```diff
@@ -234,6 +234,30 @@ describe('<Dropdown.Menu>', () => {
     assert.isNotNull(instance.querySelector('.rs-dropdown-item-submenu'));
   });
 
+  it('Should call onSelect callback with correct `eventKey` once', () => {
+    const onSelectSpy = sinon.spy();
+
+    const { getByTestId } = render(
+      <DropdownMenu onSelect={onSelectSpy}>
+        <DropdownItem data-testid="item-1" eventKey={1}>
+          1
+        </DropdownItem>
+        <DropdownItem data-testid="item-2" eventKey={2}>
+          2
+        </DropdownItem>
+        <DropdownItem data-testid="item-3" eventKey={3}>
+          3
+        </DropdownItem>
+      </DropdownMenu>
+    );
+
+    act(() => {
+      userEvent.click(getByTestId('item-1'));
+    });
```

Comment:
`userEvent` methods does `act()` automatically. You can omit this explicit `act()` call

Reply:
Done